test/src/named_sections/parsers/brackets_start_end_test.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:code_text_field/src/named_sections/named_section.dart';
+import 'package:code_text_field/src/named_sections/parsers/brackets_start_end.dart';
+import 'package:code_text_field/src/single_line_comments/single_line_comment.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('BracketsStartEndNamedSectionParser', () {
+    const comments = [
+      //
```

Comment:
is this intentional?

Reply:
Yes, any comment in a list forces the dart formatter to preserve blank lines between list items.